src/main/java/anna_gontari/Inheritance/Dog.java

```diff
@@ -0,0 +1,31 @@
+package anna_gontari.Inheritance;
+
+//Implement a class called "Animal" with instance variables for name, age, and sound.
+// Create a subclass called "Dog" that inherits from "Animal" and includes additional instance variables
+// such as breed and weight, create constructors to initialize all variables.
+public class Dog extends Animal {
+    private String breed;
+    private int weight;
+    Dog(String breed, String name, int age, int weight, String sound) {
+        super(name, age, sound);
+        this.breed = breed;
+        this.weight = weight;
+    }
+    public Dog(String name, int age, String sound, String breed, int weight) {
```

Comment:
please organize constructors in such way when will be clear the difference between them.
Because right now I see two constructors with approximatively the same parameters but with different place in constructor.

Ex 
Dog(,,,String bread)
Dog(String bread,,,,)
it is incorrect

Create a constructor with no parameters
with just one single parameter 
and wil all parameters